corehq/apps/sms/models.py

```diff
@@ -1069,6 +1069,7 @@ class MessagingEvent(models.Model, MessagingStatusMixin):
     ERROR_FCM_NO_ACTION = "FCM_NO_ACTION"
     ERROR_FCM_NOTIFICATION_FAILURE = "FCM_NOTIFICATION_FAILURE"
     ERROR_FCM_DOMAIN_NOT_ENABLED = 'FCM_DOMAIN_NOT_ENABLED'
+    ERROR_FILTER_MISMATCH = 'FILTER_MISMATCH'
```

Comment:
From briefly looking, I don't think it's needed here, but I noticed these errors have a corresponding `ERROR_MESSAGES` below. Just noting that

Reply:
I think what you have here is correct. What I meant is - I see the other errors have a corresponding key in the `ERROR_MESSAGES` dictionary on line 1074. I was wondering if `ERROR_FILTER_MISMATCH` needs to also be added to that dictionary.

Reply:
Added a message to the dictionary